www/src/components/LayoutGenerator.tsx

```diff
@@ -13,59 +13,61 @@ export interface IColumn {
   onChangeOffset: Function,
 }
 
-const Column = ({
+function Column({
   index, width, onChangeWidth, offset, onChangeOffset,
-}: IColumn) => (
-  <div
-    className={classNames('col mb-4', {
-      [`col-${width}`]: width > 0,
-      [`offset-${offset}`]: offset > 0,
-    })}
-  >
+}: IColumn) {
+  return (
     <div
-      className="text-align-center p-1"
-      style={{ background: '#eee', minHeight: '2rem' }}
+      className={classNames('col mb-4', {
+        [`col-${width}`]: width > 0,
+        [`offset-${offset}`]: offset > 0,
+      })}
     >
-      <div className="form-inline m-2">
-        <label className="font-weight-normal" htmlFor={`column-${index}-width`}>
-          Width
-        </label>
-        <Input
-          type="number"
-          id={`column-${index}-width`}
-          className="form-control-sm"
-          value={width}
-          placeholder="Width (1 - 12)"
-          style={{ width: '3rem' }}
-          min={0}
-          step={1}
-          max={12}
-          onChange={(e: React.ChangeEvent<HTMLInputElement>) => onChangeWidth(index, e.target.value)}
-        />
-      </div>
-      <div className="form-inline m-2">
-        <label
-          className="font-weight-normal"
-          htmlFor={`column-${index}-offset`}
-        >
-          Offset
-        </label>
-        <Input
-          type="number"
-          id={`column-${index}-offset`}
-          className="form-control-sm"
-          value={offset}
-          placeholder="Offset (1 - 11)"
-          style={{ width: '3rem' }}
-          min={0}
-          step={1}
-          max={11}
-          onChange={(e: React.ChangeEvent<HTMLInputElement>) => onChangeOffset(index, e.target.value)}
-        />
+      <div
+        className="text-align-center p-1"
+        style={{ background: '#eee', minHeight: '2rem' }}
+      >
+        <div className="form-inline m-2">
+          <label className="font-weight-normal mr-2" htmlFor={`column-${index}-width`}>
+            Width
+          </label>
+          <Input
+            type="number"
+            id={`column-${index}-width`}
+            className="form-control-sm"
+            value={width}
+            placeholder="Width (1 - 12)"
+            style={{ width: '4rem' }}
+            min={0}
+            step={1}
+            max={12}
+            onChange={(e: React.ChangeEvent<HTMLInputElement>) => onChangeWidth(index, e.target.value)}
+          />
+        </div>
+        <div className="form-inline m-2">
+          <label
+            className="font-weight-normal mr-2"
+            htmlFor={`column-${index}-offset`}
+          >
+            Offset
+          </label>
+          <Input
```

Comment:
[curious] Since the `Input` component is [deprecated](https://paragon-openedx.netlify.app/components/input/), should we migrate from `Input` to `Form.Control` or otherwise here?

Reply:
Done